NorthstarDLL/printmaps.cpp

```diff
@@ -109,7 +109,7 @@ void RefreshMapList()
 
 // clang-format off
 AUTOHOOK(_Host_Map_f_CompletionFunc, engine.dll + 0x161AE0,
-int, __fastcall, (const char const* cmdname, const char const* partial, char commands[COMMAND_COMPLETION_MAXITEMS][COMMAND_COMPLETION_ITEM_LENGTH]))
+int, __fastcall, (const char * cmdname, const char * partial, char commands[COMMAND_COMPLETION_MAXITEMS][COMMAND_COMPLETION_ITEM_LENGTH]))
```

Comment:
`const char const*` is a valid type hint and this is throwing away pointer-to-const compiler hinting

Reply:
clang-tidy indicates the extra `const` doesn't do anything.

Reply:
Ah i see, whoever wrote this specific function got the order of their keywords messed up. What i *assume* to be intended was `const char* const`. In its current form, this is indeed incorrect

Reply:
I agree, that probably was the intention. Would you like me to change it to `const char* const`?

Reply:
That's probably the best course of action here. I have another PR waiting for some other stuff that fixes a whole bunch of little things like this, but that needs to wait a bit for when the toolchain shift is done